fcrepo-export/src/main/java/org/fcrepo/exporter/Exporter.java

```diff
@@ -163,4 +169,21 @@ private File fileForContainer(final URI uri) {
         return new File(config.getDescriptionDirectory(), uri.getPath().replaceAll(":", "_")
             + config.getRdfExtension());
     }
+
+    private void checkValidResponse(final FcrepoResponse response, final URI uri) {
```

Comment:
I know this is `private`, but it might be nice to add the briefest doc here, just `@throws RuntimeException when authentication or authorization conditions fail` or the like.